src/app/page.js

```diff
@@ -1,5 +1,31 @@
-import Link from "next/link";
+import { FeedItem } from "@/components/feed"
+
 
 export default function Home() {
-  return <div>Feed</div>;
+
+      const items = [{
+        id: 1,
+        avatar: "https://ionicframework.com/docs/img/demos/avatar.svg",
+        username: "Osgood",
+        handle: "@osgoodpaekid",
+        createdAt: new Date('2023-07-07'),
```

Comment:
as mentioned before, this is a date object. You should use iso string

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString